site/templates/homepage.tmpl

```diff
@@ -0,0 +1,8 @@
+<%inherit file="base.tmpl"/>
+
+<%block name="front_page">
+  <div>
+    ${post.text()}
+  </div>
```

Comment:
I looks like instead of removing Footer from the homepage, you removed it from the template instead? :worried:

Reply:
Ah, I see, `base.tmpl` has `front_page` as a separate block.